go/vt/vtgate/vindexes/lookup_internal.go

```diff
@@ -304,7 +303,7 @@ nextRow:
 	// We only need to check the first row. Number of cols per row
 	// is guaranteed by the engine to be uniform.
 	if len(trimmedRowsCols[0]) != len(lkp.FromColumns) {
-		return fmt.Errorf("lookup.Create: column vindex count does not match the columns in the lookup: %d vs %v", len(trimmedRowsCols[0]), lkp.FromColumns)
+		return vterrors.Wrapf(vterrors.VT03006(), "lookup.Create: (columns, count): (%v, %d)", lkp.FromColumns, len(trimmedRowsCols[0]))
```

Comment:
I don't think we should be wrapping errors in the same place as they are created